From 34efb9a74cfd180bd9e7b5703c279f00b34619cf Mon Sep 17 00:00:00 2001
From: xc-racer99 <xc-racer2@live.ca>
Date: Sun, 1 Jan 2017 12:15:55 -0800
Subject: [PATCH] Revert "audio: Make getPresentationPosition pure virtual"

Breaks the build for me...
This reverts commit be23ef293971796afa1a58cf2232f8cd3a21bd43.
---
 include/hardware_legacy/AudioHardwareInterface.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/hardware_legacy/AudioHardwareInterface.h b/include/hardware_legacy/AudioHardwareInterface.h
index b08fece..6328063 100644
--- a/include/hardware_legacy/AudioHardwareInterface.h
+++ b/include/hardware_legacy/AudioHardwareInterface.h
@@ -120,7 +120,7 @@ public:
     /**
      * Return a recent count of the number of audio frames presented to an external observer.
      */
-    virtual status_t    getPresentationPosition(uint64_t *frames, struct timespec *timestamp) = 0;
+    virtual status_t    getPresentationPosition(uint64_t *frames, struct timespec *timestamp);
 
 };
 
-- 
2.7.4

